tronbyt_server/templates/manager/index.html

```diff
@@ -195,88 +213,89 @@ <h1>{{ device['name'] }}</h1>
         </tr>
         <tr>
           <td>
-            <button id="toggleBtn-{{ device['id'] }}" class="w3-button w3-teal w3-round" onclick="toggleDetails('{{ device['id'] }}')">{{ _('Show Details') }}</button>
```

Comment:
It's not mentioned in the PR description, so just double checking: was it intended to remove the device details from the index page?